.github/workflows/build_rocks.yaml

```diff
@@ -224,3 +226,23 @@ jobs:
         env:
           TRIVY_USERNAME: ${{ github.actor }}
           TRIVY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check trivyignore
+        run: |
+          curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.46.0
+          if [ -f ".trivyignore" ]
+          then
+            output=$(trivy image $ROCK_IMAGE --severity HIGH,CRITICAL -q -f json --ignorefile "" | jq -r '.Results[].Vulnerabilities[].VulnerabilityID' 2>/dev/null || echo "No vulnerabilities found")
```

Comment:
Isn't the severity configurable via config file? If so, we should probably take it into consideration

Reply:
The Run Github Trivy Image Action has this configured, so I used it as well:
```
354094af (arturo-seijas                2023-02-20 16:21:56 +0100 225)           severity: 'CRITICAL,HIGH'
```
I can remove it though, as we can scan for all severities here, it won't matter - if they are in the trivyignore, we'll annotate them.

Reply:
I'd rather remove them, keeping maintainability in mind

Reply:
I'll open a story about this in the backlog, I want to have a quick alignment on how we configure both actions, the one introduced here is more a no-op, I'm more interested in seeing about the other.

Reply:
Resolving to unblock the PR, though if you mind, please re-open it, that's fine with me.